src/@chakra-ui/gatsby-plugin/foundations/shadows.ts

```diff
@@ -2,6 +2,8 @@ const shadows = {
   // using css variables bc shadows do not support color tokens yet
   primary: "4px 4px 0px 0px var(--eth-colors-primaryLight)",
   outline: "0 0 0 4px var(--eth-colors-primaryHover)",
+  tableBoxShadow:
```

Comment:
lets remove the suffix since it is redundant
```suggestion
  table:
```

Reply:
Done